### PR TITLE
fix: rm-serde-arbitrary-precision-feat

### PR DESCRIPTION
**Pull request by shariffdev**
Closes https://github.com/near/workspaces-rs/issues/279

The `arbitrary_precision` feature is removed from serde_json in the Cargo.toml

### CODE REVIEW

**Comment by itsyaasir** [comment]
@frol This is ready to be reviewed

**Comment by itsyaasir** [comment]
@frol  This PR is ready to be reviewed

**Comment by shariffdev** [comment]
@frol This is ready to be reviewed.

**Comment by frol** [comment]
@shariffdev Why was it used in the first place? Does it break anything when this feature is dropped? cc @ChaoticTempest

**Comment by shariffdev** [comment]
> @shariffdev Why was it used in the first place? Does it break anything when this feature is dropped? cc @ChaoticTempest

I did not find any example that broke without the feature. Please advise @ChaoticTempest

**Review by frol — CHANGES_REQUESTED** [review]
`arbitrary_precision` was introduced in #35. Please, run ref-finance example with this PR applied and confirm that it still works just fine, and let's merge it.

**Comment by shariffdev** [comment]
I have pushed a small fix, everything is looking good now!

File: examples/src/ref_finance.rs [hunk 1]
```diff
@@ -71,7 +71,7 @@ async fn create_wnear(owner: &Account, worker: &Worker<Sandbox>) -> anyhow::Resu
         .call(wnear.id(), "new")
         .args_json(json!({
             "owner_id": owner.id(),
-            "total_supply": parse_near!("1,000,000,000 N"),
+            "total_supply": parse_near!("1,000,000,000 N").to_string(),
         }))
```
Comment (frol) [inline examples/src/ref_finance.rs:75]: `wrap.near` contract `new` method does not expect any arguments O_o

https://github.com/near/core-contracts/blob/3f3170fce91ff4d8c6ee9d15683f2d4dfe1275cf/w-near/src/lib.rs#L36

Let's remove `args_json` completely.

Comment (shariffdev) [reply examples/src/ref_finance.rs:75]: OK, let me try!

Comment (shariffdev) [reply examples/src/ref_finance.rs]: It works okay!

**Review by frol — APPROVED** [review]

**Pull request merged**